compiler-rt/lib/profile/InstrProfiling.c

```diff
@@ -89,3 +89,7 @@ COMPILER_RT_VISIBILITY void __llvm_profile_reset_counters(void) {
   }
   lprofSetProfileDumped(0);
 }
+
+inline int hasCorrelation() {
```

Comment:
The function should have a hidden visibility (the `COMPILER_RT_VISIBILITY` macro) so it's hidden from ABI.

Reply:
COMPILER_RT_VISIBILITY is added at https://github.com/llvm/llvm-project/commit/56e205a89cbb114750f2bd3f5cfbd19e209d018a.